manifest-json-schema.json

```diff
@@ -69,6 +69,12 @@
       "default": "1.x",
       "description": "The major version of OTP that is being used. Must be either `1.x` or `2.x`. This is used in order to generate the appropriate command line parameters to run OTP with."
     },
+    "peliasCsv": {
```

Comment:
I would rename to `peliasCsvUris` to be consistent with the other uri fields and to denote the field expects an array. What do others think @landonreed @evansiroky?